### PR TITLE
Add suite_artic (currently only artic_minion)

### PR DESCRIPTION
**Pull request by dfornika**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by dfornika** [comment]
I've hard-coded use of the `--medaka` variant caller to avoid the complication of needing access to `.fast5` files to run the default `nanopolish` mode for this tool.

File: tools/artic/artic_minion.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              --bwa
+            #end if
+            'name/version'
+            '${input_read_file.name}'
+    ]]></command>
+    <inputs>
+        <param name="input_read_file" type="data" format="fastq,fastqsanger,fastq.gz,fastqsanger.gz" label="Input Read File"/>
+        <param name="normalise" type="integer" min="1" value="200" max="2000" label="Coverage normalisation depth"/>
+        <param name="use_bwa" type="boolean" label="Use bwa aligner"/>
+        <param name="reference" type="data" format="fasta" label="Reference Sequence"/>
```
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/artic/artic_minion.xml:28]: Or maybe question :)

[the remark this answers is not in the document]
Comment (dfornika) [reply tools/artic/artic_minion.xml:28]: Yes, I'll do that.

[the remark this answers is not in the document]
Comment (dfornika) [reply tools/artic/artic_minion.xml:28]: What is the name of the data table that needs to be read from?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/artic/artic_minion.xml:28]: all_fasta ... copy and paste ready here: https://github.com/galaxyproject/tools-iuc/blob/8908da9cdd112ae0943dbf1eccb221e84cd99ca7/tools/scpipe/scpipe.xml#L121

wth tests included ...

**Comment by bgruening** [comment]
I found this here: 
> |  error: FASTA reference file must be indexed with samtools faidx. Index file should have same name as FASTA file with .fai appended.

**Comment by dfornika** [comment]
Tests 1 and 3 are passing but test 2 is failing on the `longshot` step:

```
Running: longshot -P 0 -F -A --no_haps --bam SRR11410539_seqtk_sample_250_1.fastq.primertrimmed.rg.sorted.bam --ref scheme/name/version/name.reference.fasta --out SRR11410539_seqtk_sample_250_1.fastq.longshot.vcf --potential_variants SRR11410539_seqtk_sample_250_1.fastq.merged.vcf.gz
Command failed:longshot -P 0 -F -A --no_haps --bam SRR11410539_seqtk_sample_250_1.fastq.primertrimmed.rg.sorted.bam --ref scheme/name/version/name.reference.fasta --out SRR11410539_seqtk_sample_250_1.fastq.longshot.vcf --potential_variants SRR11410539_seqtk_sample_250_1.fastq.merged.vcf.gz
```

I'll have to take a closer look but I'm a bit puzzled because the only difference is that the primer scheme .bed file is being read from the history (instead of a tool data table) in Test 2.

File: tools/artic/artic_minion.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
             '${read_file.element_identifier}'
     ]]></command>
     <inputs>
-        <param argument="--read-file" type="data" format="fastq,fastqsanger,fastq.gz,fastqsanger.gz" label="Input Read File"/>
+        <param name="read_file" argument="--read-file" type="data" format="fastq,fastqsanger,fastq.gz,fastqsanger.gz" label="Input Read File"/>
```
Comment (bgruening) [inline tools/artic/artic_minion.xml:31]: that should work without name, Galaxy will create read_file automatically.

**Comment by dfornika** [comment]
The ARTIC project provides two .bed files for their primer schemes:

https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V3

It seems that there's an issue with the `.scheme.bed` version when it's loaded from the history. I've included one of each in the `test-data`, and I'm using the six-column `.bed` version for the test that loads the scheme from the history.

**Comment by bgruening** [comment]
That is a pretty great start. Thanks @dfornika!

**Pull request merged**